src/main/resources/migrations/0012_alter_table_webhook_messages_add_last_delivery_status.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE webhook_messages
+ADD COLUMN last_delivery_status VARCHAR(64) check (last_delivery_status in ('SUCCESSFUL', 'FAILED', 'PENDING', 'WILL_NOT_SEND'));
```

Comment:
Do we need to check the list of values here? Adding a new enum value to the code could cause issues if there was no equivalent db migration. or at least we should have a test to check that db inserts still work

Reply:
https://github.com/alphagov/pay-webhooks/commit/4b6553b8dc2c934d67ee96b503bf394faa653f0e